auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/config/UmsClientConfig.java

```diff
@@ -23,6 +23,9 @@
     @Value("${altus.ums.client.list_service_principal_cloud_identities_page_size:100}")
     private int listServicePrincipalCloudIdentitiesPageSize;
 
+    @Value("${altus.ums.calling_service_name:cloudbreak}")
```

Comment:
spring property with underscores looks strange, I would use `caller.service.name` or simply `caller`, what do you think about it?

Reply:
Renamed the property to `caller`.